springfield/firefox/views.py

```diff
@@ -599,3 +599,24 @@ def get_template_names(self):
             template_name = "firefox/features/pdf-editor.html"
 
         return [template_name]
+
+
+class firefox_features_adblocker(L10nTemplateView):
```

Comment:
style nit: For views that are classes, the convention is to use PascalCase rather than the snake_case we use for function names.